tools/pull-request-labeler/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  name: 'pr-labeler',
```

Comment:
NIT: Maybe use the same name as used in nx.json and angular.json (tools-pull-request-labeler) for consistency